src/PlaywrightSharp.Tests/Page/PageOpenerTests.cs

```diff
@@ -0,0 +1,45 @@
+using System.Threading.Tasks;
+using PlaywrightSharp.Tests.BaseTests;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace PlaywrightSharp.Tests.Page
+{
+    ///<playwright-file>page.spec.js</playwright-file>
+    ///<playwright-describe>Page.opener</playwright-describe>
+    public class PageOpenerTests : PlaywrightSharpPageBaseTest
+    {
+        internal PageOpenerTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        ///<playwright-file>page.spec.js</playwright-file>
+        ///<playwright-describe>Page.opener</playwright-describe>
+        ///<playwright-it>should provide access to the opener page</playwright-it>
+        [Fact]
+        public async Task ShouldProvideAccessToTheOpenerPage()
+        {
+            var (popupEvent, _) = await TaskUtils.WhenAll(

```

Comment:
WhenAll doesn't return all the tasks as an array like Promise.all in javascript.

Reply:
is this relevant? we are returning a tuple here after calling `Task.WhenAll` on all of the tasks

Reply:
not relevant.